### PR TITLE
Labels are now added to model config under id2label and label2id

### LINKED ISSUE #2487: "config.json" does not include correct "id2label" and "label2id" after finetuning on NER task
## 🐛 Bug

Model I am using (Bert, XLNet....): xlmroberta

Language I am using the model on (English, Chinese....): English

The problem arise when using:
* [ ] the official example scripts: (give details)
I use script `run_ner.py` in order to finetune xlmroberta on conll03 dataset.

The script executed with no problems. But the file "config.json" in the output directory is not correct.

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
Task: NER
Dataset:  conll03

## To Reproduce

1.
I run the script: run_ner.py as follows:
`python run_ner.py --data_dir 0-data --model_type 'xlmroberta' --model_name_or_path 'xlm-roberta-large' --output_dir 1-out --max_seq_length 32 --do_train --do_eval --per_gpu_train_batch_size 8 --no_cuda --evaluate_during_training --logging_steps 1756 --save_steps 1756 --eval_all_checkpoints`
2. Go to the output directory. The file "config.json"  contains :
"label2id": {
    "LABEL_0": 0,
    "LABEL_1": 1
  },
and
"id2label": {
    "0": "LABEL_0",
    "1": "LABEL_1"
  },
which are not expected in NER
3.

## Expected behavior

I expect that "config.json" contains something like:
"id2label": {
    "0": "B-LOC",
    "1": "B-MISC",
    "2": "B-ORG",
    "3": "I-LOC",
    "4": "I-MISC",
    "5": "I-ORG",
    "6": "I-PER",
    "7": "O"
  },
and
"label2id": {
    "B-LOC": 0,
    "B-MISC": 1,
    "B-ORG": 2,
    "I-LOC": 3,
    "I-MISC": 4,
    "I-ORG": 5,
    "I-PER": 6,
    "O": 7
  },
## Environment

* OS: Ubuntu 18.04
* Python version: 3.6.9
* PyTorch version: 1.3.1
* PyTorch Transformers version (or branch): 2.3.0
* Using GPU ? No
* Distributed or parallel setup ? No
* Any other relevant information:

### PR DESCRIPTION
**Pull request by marma**
Fixes huggingface/transformers#2487

### CODE REVIEW

**Comment by julien-c** [comment]
I had noticed this indeed, thanks for fixing!

**Comment by naveenjafer** [comment]
@marma So the labels are supposed to be the actual labels instead of placeholders like "Label_0", "Label_1" ... right? I tried it out in 3.3.1 but it still generates the placeholder labels in config. I guess I am missing something?

**Comment by konstantinmiller** [comment]
I would also be interested in learning how I can store the actualy label2id and id2label dictionaries along a pre-trained model. Is that possible?

**Comment by cceyda** [comment]
@konstantinmiller & @naveenjafer You can pass label2id and id2label to config then pass that config to model like in below snippet:
```py
config = AutoConfig.from_pretrained(
        model_args.config_name if model_args.config_name else model_args.model_name_or_path,
        num_labels=num_labels,
        id2label={i: label for i, label in enumerate(label_list)},
        label2id={label: i for i, label in enumerate(label_list)},
        finetuning_task=data_args.task_name)
model = AutoModelForSequenceClassification.from_pretrained(
        model_args.model_name_or_path,
        from_tf=bool(".ckpt" in model_args.model_name_or_path),
        config=config )
```
